packages/e2e/example/test_driver/example_e2e_test.dart

```diff
@@ -1,12 +1,22 @@
 import 'dart:async';
 import 'dart:io';
 
+import 'package:e2e/common.dart' as common;
 import 'package:flutter_driver/flutter_driver.dart';
 
 Future<void> main() async {
   final FlutterDriver driver = await FlutterDriver.connect();
-  final String result =
+  final String jsonResult =
       await driver.requestData(null, timeout: const Duration(minutes: 1));
+  final common.Response response = common.Response.fromJson(jsonResult);
   await driver.close();
-  exit(result == 'pass' ? 0 : 1);
+
+  if (response.allTestsPassed) {
```

Comment:
It seems like we should have this match the README.

Reply:
changed the code and updated the README.